outputsheets/d20/4e/pdf/common_sheet/block_weapons.xslt

```diff
@@ -744,7 +744,7 @@
 									<xsl:with-param name="attribute" select="'weapon.title'"/>
 							</xsl:call-template>
 								<fo:block font-size="7pt" padding-right="2pt">
-									<xsl:value-of select="number($distance)"/>
```

Comment:
I had some reasoning for this and completely forgot. Going to close this for now